dbhydra/src/tables.py

```diff
@@ -725,12 +725,12 @@ def get_num_of_records(self):
         Returns the number of records in table
         """
 
-        num_of_records = self.select(f"SELECT COUNT(*) FROM {self.name};")
+        num_of_records = self.select(f"SELECT COUNT(*) FROM `{self.name}`;")
```

Comment:
shouldnt here be "{quote}"?

Reply:
Could be `self.db1.identifier_quote` but in concrete classes I was using specific quotes to decrease verbosity.